roles/openshift_metrics/tasks/main.yaml

```diff
@@ -1,13 +1,13 @@
 ---
-- local_action: shell python -c 'import passlib' 2>/dev/null || echo not installed
+- name: Check that python-passlib is available on the control host
+  local_action: htpasswd path=/dev/null name=dummy state=present password=dummy
   register: passlib_result
   become: false
+  ignore_errors: yes
 
-- name: Check that python-passlib is available on the control host
-  assert:
-    that:
-      - "'not installed' not in passlib_result.stdout"
-    msg: "python-passlib rpm must be installed on control host"
+- fail:
```

Comment:
Can you actually rewrite this so that it uses `assert` instead? This way we can avoid seeing a `skipped` message when we don't fail vs nothing being printed out.